app/src/androidTest/java/sdp/moneyrun/LeaderboardInstrumentedTest.java

```diff
@@ -35,7 +35,7 @@ public void addPlayerWorks(){
                 player.setScore(8008);
                 a.addPlayer(player);
 
-                assertEquals(a.getPlayerList().size(),1);
+                assertEquals(a.getPlayerList().size(),6);
```

Comment:
What is "a" in this lambda and why is the player list 6 when only one was added ?

Reply:
a represents the Leaderborad activity instance and the list is 6 since when a player logs in dummy players are going to be added to the leaderboard with attached listeners to them, the idea is once the game starts these dummy players turn into actual players  and have the leaderboard change accordingly with the game